Model/Address/Factory/AddressFactory.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SyliusConsumerBundle\Model\Address\Factory;
+
+use Sulu\Bundle\SyliusConsumerBundle\Model\Address\Address;
+use Sulu\Bundle\SyliusConsumerBundle\Model\Address\AddressInterface;
+use Sulu\Bundle\SyliusConsumerBundle\Model\PayloadTrait;
+
+class AddressFactory
+{
+    use PayloadTrait;
+
+    public function createFromArray(array $data): AddressInterface
+    {
+        $this->initializePayload($data);
```

Comment:
good idea? this is a state on a "singleton"?

Reply:
yep.. should be changed in future